.env.sample

```diff
@@ -0,0 +1,5 @@
+NODE_ENV=development
+MONGO_URI=mongodb://localhost:27017/codinggardencommunity
```

Comment:
Should be a deployed development/testing database, so tests doesn't fail when a databcase connection cannot made.

Reply:
Docker is generally a good option for CI and test databases